test/people/CMakeLists.txt

```diff
@@ -15,5 +15,5 @@ endif()
 include_directories(SYSTEM ${VTK_INCLUDE_DIRS})
 PCL_ADD_TEST(a_people_detection_test test_people_detection
              FILES test_people_groundBasedPeopleDetectionApp.cpp
-             LINK_WITH pcl_gtest pcl_common pcl_io pcl_kdtree pcl_search pcl_features pcl_sample_consensus pcl_filters pcl_io pcl_segmentation pcl_people
```

Comment:
The test uses `pcl::io::loadPCDFile`, so not linking to `pcl_io` seems wrong?

Reply:
We still do. It was a dublicate.

Reply:
Ah, didn't see that :+1: